lib/salus/scanners/slither.rb

```diff
@@ -82,6 +82,12 @@ def config_opts
         #       A list of paths must be joined by | as in a regular expression, not comma
         opts += ' --filter-paths ' + @config['filter-paths'].to_s + ''
       end
+      if !@config.key?('exclude-informational') || @config['exclude-informational']
```

Comment:
can you use `dig` instead?

Reply:
Also, will your code behavior differently if
```yaml
exclude-information: "false"
```

vs

```yaml
exclude-information: false
```

Reply:
Yes `"false"` and `false` will cause different behavior since the first truthy and the second is falsey. Do we want to support multiple types for config options? I assumed we only wanted to accept boolean values for these options.

Reply:
Also, I think a `dig` call would work here if the condition were `@config.key?('exclude-informational') || @config['exclude-informational']`, but I think the fact that it's `!@config.key?('exclude-informational') || @config['exclude-informational']` (note the bang) makes this condition semantically different from `dig`.